### ISSUE TITLE
Status update

### ISSUE THREAD

**Issue by wrgrant** [F0rged/LittleWheelsESC#1]
Hey, looks like its been a bit since you touched this.

I have a similar (albeit much simpler) project [over here](https://gitlab.com/wrgrant/toy-car-control/-/blob/master/toy-car-control.ino).

It would be super fun to have a fellow diy-kids-toy-maker to bounce ideas off of.

Interested?

**Comment by F0rged** [comment]
Absolutely!  Just had my hot water tank explode last night so am pretty busy right now but would love to collaborate once the dust has settled

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
